### ISSUE TITLE
[YSQL] Enable cloud admin user to create orafce extension

### ISSUE THREAD

**Issue by paullee-yb** [yugabyte/yugabyte-db#12509]
Jira Link: [[DB-344]](https://yugabyte.atlassian.net/browse/DB-344)
### Description

```
yugabyte=> CREATE EXTENSION IF NOT EXISTS Orafce;
ERROR:  permission denied for schema pg_catalog
```

[DB-344]: https://yugabyte.atlassian.net/browse/DB-344?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

**Comment by paullee-yb** [comment]
Creating orafce extension in YB Managed (2.15.0) still does not work:
```
yugabyte=> CREATE EXTENSION orafce;
ERROR:  permission denied for schema pg_catalog
yugabyte=> \du
                                                                   List of roles
  Role name   |                         Attributes                         |                               Member of                               
--------------+------------------------------------------------------------+-----------------------------------------------------------------------
 admin        | Create role, Create DB, Bypass RLS                         | {yb_superuser}
 postgres     | Superuser, Create role, Create DB, Replication, Bypass RLS | {}
 yb_db_admin  | No inheritance, Cannot login                               | {}
 yb_extension | Cannot login                                               | {}
 yb_fdw       | Cannot login                                               | {}
 yb_superuser | Create role, Create DB, Cannot login, Bypass RLS           | {pg_read_all_stats,pg_signal_backend,yb_extension,yb_fdw,yb_db_admin}
 yugabyte     | Superuser, Create role, Create DB, Replication, Bypass RLS | {}

yugabyte=> CREATE EXTENSION orafce;
ERROR:  permission denied for schema pg_catalog
```

**Comment by paullee-yb** [comment]
Change was landed in 2.15.1, so this is expected.
